src/Options.php

```diff
@@ -696,14 +696,6 @@ private function normalizeAbsolutePath($value)
             $path = $value;
         }
 
-        if (
-            \DIRECTORY_SEPARATOR === substr($path, 0, 1)
-            && \DIRECTORY_SEPARATOR !== substr($path, -1)
-            && '.php' !== substr($path, -4)
-        ) {
-            $path .= \DIRECTORY_SEPARATOR;
```

Comment:
Removing this is a behavioral change. Since I don't think the server is gonna care about this, we can ignore trailing slashes here.